public/index.html

```diff
@@ -60,13 +60,32 @@ <h2 style="text-align: center;">🧪newman-server🧪</h2>
             <button type="submit" id="run-button" class="btn btn-primary">Run the collection</button>
             <div id="run-button-description" class="form-text"></div>
         </form>
+
+        <form id='newman-convert-form' method='post' encType="multipart/form-data" action="/convert/html">
+            <div class="mb-3">
+                <label for="formFile" class="form-label"><strong>Collection summary file</strong></label>
+                <input class="form-control" type="file" name="summaryFile" accept=".json" required>
+                <div class="form-text">
+                    Once you've ran your collection in Newman with the output format selected to be JSON, you will receive the 
+                    run summary. Add the summary to a JSON file and this is the file that will be used by newman to convert to HTML report.
+                </div>
```

Comment:
can you move that in a dedicated div?

Currently it make it hard to see where the first form ends and the second form starts

![image](https://user-images.githubusercontent.com/18073057/187915440-e606d7b4-8e36-499d-8960-26bc582b9abe.png)

Reply:
okay on it.